src/Controller/PatternLibraryController.php

```diff
@@ -74,11 +77,12 @@ public function single($name) {
   public function overview() {
 
     $definitions = $this->patternsManager->getDefinitions();
-    foreach ($definitions as $name => $definition) {
-      $definitions[$name]['rendered']['#type'] = 'pattern_preview';
-      $definitions[$name]['rendered']['#id'] = $name;
-      $definitions[$name]['meta']['#theme'] = 'patterns_meta_information';
-      $definitions[$name]['meta']['#pattern'] = $definition;
+    foreach ($definitions as $definition) {
```

Comment:
I'm looking into [Kernel tests](https://www.drupal.org/docs/8/phpunit/running-phpunit-tests) for this part as we would really need the container to be setup and running at this point.